x-pack/plugins/reporting/server/core.ts

```diff
@@ -184,12 +195,9 @@ export class ReportingCore {
   private async assertKibanaIsAvailable(): Promise<void> {
     const { status } = this.getPluginSetupDeps();
 
-    await status.overall$
-      .pipe(
-        filter((current) => current.level === ServiceStatusLevels.available),
-        first()
-      )
-      .toPromise();
+    await Rx.firstValueFrom(
+      status.overall$.pipe(filter((current) => current.level === ServiceStatusLevels.available))
+    );
```

Comment:
Let's keep this expression the same as in `main`, because it is part of an investigation with stability issues at the moment.